plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/project/ManageProjectsWizard.java

```diff
@@ -30,10 +31,13 @@ public ManageProjectsWizard(Connection connection) {
 		this(null, connection);
 	}
 
+	public ManageProjectsWizard(IProject project) {
```

Comment:
what's the point of adding this constructor?

Reply:
@bdshadow if a project is selected, the wizard will derive the connection from the project. In the other case a connection is selected, so there's no need to derive the connection.

Reply:
yes, i see it. But i haven't found any usage of this constructor. I even tried to comment this lines - works fine. So it looks to me like no need in new lines of code

Reply:
@bdshadow ok, removed it